README.md

```diff
@@ -42,29 +51,79 @@ Value is X-JWT-Assertion-Original.
 
 Example
 
-```
-import byu_jwt
-original_jwt_header = byu_jwt.BYU_JWT_HEADER_ORIGINAL
+```python
+original_jwt_header = byujwt.BYU_JWT_HEADER_ORIGINAL
 ```
 
-## Testing
-
-You will need a file named ~/.byu/byu-jwt-python.yaml with the following contents. Get your info at https://api.byu.edu/store/site/pages/subscriptions.jag
-```
-client_id: <your WSO2 application's Consumer Key>
-client_secret: <your WSO2 application's Consumer Secret>
-```
+### Example Python Lambda function that makes use of caching
+```python
+import byu_jwt
 
-Make sure you have python and python3 installed on your system, then install virtualenv in whatever way you install python modules (usually `$ pip install virtualenv`).
+byujwt = byu_jwt.JWT()
 
+def handler(event, context):
+    jwt_to_decode = event['headers']['X-JWT-Assertion']
+    try:
+        jwt = byujwt.decode(jwt_to_decode)
```

Comment:
The example might be better if we use the JWT here to do something like return "Hello " + first name or something.